Sources/Tuvi.Core.Mail.Impl/Commands.cs

```diff
@@ -196,7 +195,7 @@ private async Task PrepareToUse(ICredentialsProvider credentialsProvider, Cancel
 
             if (!Service.IsAuthentificated)
             {
-                await Service.AuthenticateAsync(credentialsProvider, cancellationToken).ConfigureAwait(false);
+                await Service.AuthenticateAsync(cancellationToken).ConfigureAwait(false);
             }
         }
 
```

Comment:
Rename `Execute` to `ExecuteAsync` (line 203)

Reply:
Done.